### ISSUE TITLE
About adding one more IF BLOCK

### ISSUE THREAD

**Issue by JasonSheng-atp** [hzwer/arXiv2020-RIFE#216]
hello，请问如果再加一个IFblock，或者在IFblock使用卷积核size=5的卷积层，对性能会有什么影响？我看ablation study没有做，自己试了一下，发现train崩了，loss越来越高。

**Comment by hzwer** [comment]
hello，我现在的认知是，Vimeo90K 包含的物体运动并不算非常大，所以再加 block 不是必要的
对于训崩的情况，简单做法是降低 learning rate 或增加 batchnorm（可能会有额外推理开销）
我自己试过 4block 的网络，比起 3block 性价比要低一点

**Comment by JasonSheng-atp** [comment]
谢谢这么快的回复，我还试了forward warp的RIFE，感觉图像边界上插得比backward warp的RIFE会好一点，总体上因为black hole的问题psnr会掉0.2左右，RIFE真的是很nice的算法。

**Comment by hzwer** [comment]
对；我觉得 forward warp 是很科学的做法，因为网络会比较容易学会估计光流和图像的边界一致，这样能把边界保留得更好；
但当网络泛化不够好的时候，black hole 会很严重
